Services/Maps/classes/class.ilOpenLayersMapGUI.php

```diff
@@ -78,7 +78,9 @@ function getHtml()
 
 
 		$lng->loadLanguageModule("maps");
-		$tpl->addJavaScript("Services/Maps/js/OpenLayers.js");
+		$tpl->addCss("libs/bower/bower_components/OpenLayers/ol.css");
+		$tpl->addJavaScript("libs/bower/bower_components/OpenLayers/ol.js");
```

Comment:
These files are missing in the diff. The bower-deps need to be added to the repo just like the composer deps. It is not sufficient to just update the bower.json since we are shipping all code via this repo.

Reply:
done